docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_while_exploring_data_in_a_notebook_without_using_a_data_context.rst

```diff
@@ -3,16 +3,227 @@
 How to create Expectations while exploring data in a notebook, without using a Data Context.
 ============================================================================================
 
-.. admonition:: Admonition from Mr. Dickens
+Building :ref:`Expectations` as you conduct exploratory data analysis is a great way to ensure that your insights about data processes and pipelines remain part of your team's knowledge.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+This guide will help you get started quickly in Great Expectations, without even setting up a :ref:`Data Context`. All you need is a notebook and some data.
 
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+    - Installed Great Expectations (e.g. ``pip install great_expectations``)
+    - Have access to a notebook (e.g. ``jupyter notebook``, ``jupyter lab``, etc.)
+    - Be able to access data from your notebook
+    - Nothing else. Unlike most how-to guides, these instructions do *not* assume that you have configured a Data Context by running ``great_expectations init``.
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+Steps
+-----
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+All of these steps take place within your notebook:
+
+1. **Import Great Expectations.**
+
+    .. code-block:: python
+
+        import great_expectations as ge
+
+2. **Load some data.**
+
+    The simplest way to do this is with ``read_csv``.
+
+    .. code-block:: python
+
+        my_df = ge.read_csv("my_data_directory/titanic.csv")
+
+    This method behaves exactly the same as ``pandas.read_csv``, so you can add parameters to parse your file:
+    
+    .. code-block:: python
+
+        my_df = ge.read_csv(
+            "my_data_directory/my_messy_data.csv",
+            sep="\t",
+            skiprows=3
+        )
+
+    Similarly wrapped versions of other pandas methods (``read_excel``, ``read_table``, ``read_parquet``, ``read_pickle``, ``read_json``, etc.) are also available. Please see the <great_expectations utils module> for details.
+
+    If you wish to load data from somewhere else (e.g. from a SQL database or blob store), please fetch a copy of the data locally, or :ref:`configure a Datasource <tutorials__getting_started__connect_to_data>`. (In the future, you will likely want to configure Datasources anyway, to take advantage of more of Great Expectations' advanced features.)
```

Comment:
I don't think the redirect should be to configure a datasource, but instead to use the raw classes (`SparkDFDataset` or `SqlAlchemyDataset`)

Reply:
Agreed. Again, I'm avoiding module docs for the moment, because they don't seem to be consistently implemented today. If there's a way to whip them into shape quickly, great. Otherwise, it's a pile of work I'm reluctant to take on right now.